app/views/candidate_interface/gcse_details/edit_type.html.erb

```diff
@@ -3,18 +3,18 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= form_with model: @application_qualification, url: candidate_interface_gcse_details_update_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+    <%= form_with model: @application_qualification, url: candidate_interface_gcse_details_update_type_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder, method: :post do |f| %>
       <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-8">
-        <h1 class="govuk-fieldset__heading"><%= heading_for_gcse_edit(@application_qualification.subject) %></h1>
+        <h1 class="govuk-fieldset__heading"><%= heading_for_gcse_edit_type(@subject) %></h1>
       </legend>
 
       <%= f.govuk_error_summary %>
 
       <%= f.govuk_radio_buttons_fieldset :qualification_type, legend: { size: 'm', text: 'Type of qualification' } do %>
-
         <% select_gcse_qualification_type_options.each do |option| %>
```

Comment:
When a candidate chooses `Other UK qualifications`, they should be able to add in a text field their type.

![image](https://user-images.githubusercontent.com/42817036/68208206-9a5f7500-ffc8-11e9-9ef0-0ac766f34765.png)

Also `Non-UK qualification` isn't on the prototype, is there a reason why we have this?

Reply:
I spoke with Rachael, a new trello card is coming soon for the 'other UK qualification'
I'll remove the Non-UK since we do not need it as you said 👍